src/plugins/intel_cpu/src/transformations/snippets/x64/pass/snippets_mark_skipped.cpp

```diff
@@ -254,7 +254,7 @@ bool isSuitableChildForFusingMatMul(const std::shared_ptr<const Node> &node, con
     ov::PartialShape matmul_shape;
     for (const auto &parent_out : node->input_values()) {
         const auto parent = parent_out.get_node_shared_ptr();
-        if (ov::op::util::is_on_constant_path(parent)) {
+        if (ov::op::util::is_on_constant_path(parent->output(0))) {
```

Comment:
Shouldn't it be `parent_out` instead?
As far as I understand there is no guarantee `parent` is connected with `node` via 0th port.

Reply:
changed to `parent_out`, thanks